Source/WebCore/html/ColorInputType.cpp

```diff
@@ -271,7 +271,7 @@ void ColorInputType::showPicker()
         if (!m_chooser)
             m_chooser = chrome->createColorChooser(*this, valueAsColor());
         else
-            m_chooser->reattachColorChooser(valueAsColor());
+            RefPtr { m_chooser }->reattachColorChooser(valueAsColor());
```

Comment:
I guess this is the case where we've been adding `protectedChooser()`. Or Maciej's suggestion of `m_chooser->protected()`. I'll bring this up on Slack again. But doesn't block your patch.